src/ContinuousEnergyNeutron.tex

```diff
@@ -110,16 +110,19 @@ \subsubsection{\textsf{ESZ} Block}\label{sec:ESZBlock}
 \end{ThreePartTable}
 
 \subsubsection{\textsf{NU} Block}\label{sec:NUBlock}
-There are three possibilities for the \Block{NU}:
+There are two possibilities for the \Block{NU}:
 \begin{enumerate}
   \item {\bfseries\sffamily No NU Block.} (\jxs{2}=0)
-  \item {\bfseries\sffamily Either prompt or total \nubar\ is given (but not both).} (\jxs{2} > 0) \\
-    The \aceArray{NU} array begins at location \xss{KNU} where \var{KNU}=\jxs{2} + 1.
-  \item {\bfseries\sffamily Both prompt and total \nubar\ are given.} (\jxs{2} < 0).
-    \begin{itemize}
-      \item The prompt \aceArray{NU} array begins at \xss{KNU} where \var{KNU}=\jxs{2} + 1. 
-      \item The total \aceArray{NU} array begins at \xss{KNU} where {\sffamily \var{KNU} = \jxs{2} + ABS(\xss{\jxs{2}})+1}.
-    \end{itemize}
+  \item {\bfseries\sffamily NU Block exists.} (\jxs{2} > 0)
+  \begin{itemize}
+	\item {\bfseries\sffamily Prompt or total \nubar\ is given (but not both).} (\xss{\jxs{2}} > 0) \\
+	The \aceArray{NU} array begins at location \xss{KNU} where \var{KNU}=\jxs{2} + 1.
```

Comment:
From what I can see in the NJOY source code (see subroutine change in acefc.f90), when either prompt or total is given, LNU is stored in ```\xss{\jxs{2}}``` so that in this case ```\var{KNU}=\jxs{2}```

Reply:
I concur with this!